mealy/error_analyzer.py

```diff
@@ -292,16 +301,21 @@ def _compute_ranking_arrays(self, n_purity_levels=ErrorAnalyzerConstants.NUMBER_
         self._quantized_impurity = np.digitize(self._impurity, purity_bins)
         self._difference = correctly_predicted_samples - wrongly_predicted_samples  # only negative numbers
 
-    def get_ranked_leaf_ids(self, leaf_selector, rank_by='purity'):
+        y = self._error_train_y
+        n_total_errors = y[y == ErrorAnalyzerConstants.WRONG_PREDICTION].shape[0]
+        self._global_error = wrongly_predicted_samples.astype(float) / n_total_errors
```

Comment:
```suggestion
        self._global_error = wrongly_predicted_samples / float(n_total_errors)
```
Less costly to convert into to float than change type of whole array